tests/manual-test-cases/Group5-Functional-Tests/5-5-Heterogeneous-ESXi.robot

```diff
@@ -37,7 +37,9 @@ Test
     Append To List  ${list}  ${esx3}
 
     # Finish vCenter deploy
-    ${output}=  Wait For Process  ${pid-vc}
+    ${output}=  Wait For Process  ${pid-vc}  timeout=40 minutes  on_timeout=terminate
```

Comment:
Then do you think if we should add this timeout mechanism to all the nightly tests where we have `wait for process` when deploying a vc, an esx, or a testbed?